src/core/utils/stringutils.cpp

```diff
@@ -66,3 +69,29 @@ bool StringUtils::fuzzyMatch( const QString &source, const QString &term )
            ? true
            : false;
 }
+
+QString StringUtils::pointInformation( const QgsPoint &point, const QgsCoordinateReferenceSystem &crs )
+{
+  QString firstSuffix;
+  QString secondSuffix;
+#if _QGIS_VERSION_INT >= 32500
+  const bool currentCrsIsXY = QgsCoordinateReferenceSystemUtils::defaultCoordinateOrderForCrs( crs ) == Qgis::CoordinateOrder::XY;
+  const QList<Qgis::CrsAxisDirection> axisList = crs.axisOrdering();
+  if ( axisList.size() >= 2 && crs.isGeographic() )
+  {
+    firstSuffix = QgsCoordinateReferenceSystemUtils::axisDirectionToAbbreviatedString( axisList.at( 0 ) );
+    secondSuffix = QgsCoordinateReferenceSystemUtils::axisDirectionToAbbreviatedString( axisList.at( 1 ) );
+  }
+#else
+  const bool currentCrsIsXY = true;
+  firstSuffix = crs.isGeographic() ? QStringLiteral( "E" ) : QString();
+  secondSuffix = crs.isGeographic() ? QStringLiteral( "N" ) : QString();
+#endif
+  // cppcheck-suppress knownConditionTrueFalse
+  const QString firstNumber = QString::number( currentCrsIsXY ? point.x() : point.y(),
```

Comment:
https://stackoverflow.com/a/34037164

Add the define to cppckecj jnstead?

Reply:
@m-kuhn , I'm not quite getting what you are hinting at here.

Reply:
I think you need to suppress the warning because `_QGIS_VERSION_INT` is not set. We'd prefer the check to be done with a recent qgis version

Reply:
The warning comes from currentCrsIsXY being defined as always true for _QGIS_VERSION_INT < 32500. I want to keep as little code as possible within these #ifdef #else #endif blocks.